build_tools/ci/run_matmul_test.sh

```diff
@@ -147,6 +148,8 @@ cd ${OUTPUT_DIR}
 export MATMUL_TESTS_RUN=0
 export MATMUL_TESTS_FAILS=0
 
+DEVICE_HAL="${DEVICE_HAL:-xrt-lite}"
```

Comment:
In order to run this test on windows, do we need to change this line so DEVICE_HAL can be xrt?

Reply:
https://github.com/nod-ai/iree-amd-aie/blob/main/.github/workflows/ci-windows.yml#L180

Reply:
On the CI computer, this link of course is a good solution. But can we make it easier for developers to run this script on a local machine?

Reply:
For the final time: XRT is deprecated so I am not spending absolutely any time documenting/fixing/describing/explaining anything about it.

> So not all developers need to read the CI script to duplicate the effort?

Developers are responsible for reading code relevant to their jobs.